src/createApp.ts

```diff
@@ -866,15 +923,19 @@ const createApp = compose({
 			return instanceRegistries.get(this).identifyStore(store);
 		},
 
-		createWidget<U extends Child, O>(this: App, factory: ComposeFactory<U, O>, options: any = {}): Promise<[ string, U ]> {
+		createWidget<U extends Child, O extends WidgetFactoryOptions>(
+			this: App,
+			factory: ComposeFactory<U, O>,
+			options: any = {}
+		): Promise<[ string, U ]> {
 			const app = this;
```

Comment:
That was https://github.com/dojo/app/pull/26 but I can clean that up in a new PR.